src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -365,7 +365,9 @@ private Task ProcessNegotiate(HttpContext context, HttpSocketOptions options, Co
             context.Response.ContentType = "application/json";
 
             // Establish the connection
-            var connection = _manager.CreateConnection();
+            var transportPipeOptions = new PipeOptions(pauseWriterThreshold: options.TransportPauseWriterThreshold, resumeWriterThreshold: options.TransportPauseWriterThreshold/2);
```

Comment:
2 things.
- Format the document
- You're still not setting the pipe options for direct websockets...